microshift_running_apps/microshift-operators.adoc

```diff
@@ -15,8 +15,4 @@ Operators offer a more localized configuration experience and integrate with Kub
 [id="how-to-install-operators_{context}"]
 == How to install Operators in {product-title}
 
-To minimize the footprint of {product-title}, Operators are installed directly with manifests instead of using the Operator Lifecycle Manager (OLM). The following examples provide instructions on how you can use the `kustomize` configuration management tool with {product-title} to deploy an application. Use the same steps to install Operators with manifests.
-
-include::modules/microshift-manifests-overview.adoc[leveloffset=+2]
-
-include::modules/microshift-applying-manifests-example.adoc[leveloffset=+2]
\ No newline at end of file
+To minimize the footprint of {product-title}, Operators are installed directly with manifests instead of using the Operator Lifecycle Manager (OLM). You can use the `kustomize` configuration management tool with {product-title} to deploy an application. Use the same steps to install Operators with manifests. Read xref:../microshift_running_apps/microshift-applications.adoc#microshift-manifests-overview_applications-microshift[Application deployment with {product-title}] for more information on manifests.
```

Comment:
```suggestion
To minimize the footprint of {product-title}, Operators are installed directly with manifests instead of using the Operator Lifecycle Manager (OLM). You can use the `kustomize` configuration management tool with {product-title} to deploy an application. Use the same steps to install Operators with manifests. Read xref:../microshift_running_apps/microshift-applications.adoc#microshift-manifests-overview_applications-microshift[Application deployment with {product-title}] for more information about manifests.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i